tree/tree/inc/TVirtualIndex.h

```diff
@@ -21,20 +21,20 @@
 //                                                                      //
 //////////////////////////////////////////////////////////////////////////
 
-
+#include <memory>
```

Comment:
Changes in TVirtualIndex.h are leftover from previous commit, will remove them after the next round of review